client/components/button/style.scss

```diff
@@ -211,6 +211,20 @@ button {
 		}
 	}
 
+	&.is-primary {
```

Comment:
This section is getting pretty long so consider breaking this out to:
```
.button.is-borderless.is-primary {
```

Mainly this is an optional readability thing. For example I had to load more line of this file to see what this was modifying.

Reply:
I agree. I had to fight the urge to refactor the whole thing for a small change. I think the example needs refactoring as well and the ability to cycle through button styles.